ray-operator/config/samples/ray-cluster.auth-manual.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: ray.io/v1
+kind: RayCluster
+metadata:
+  name: ray-cluster-with-manual-auth
```

Comment:
In the docs we will show how to enable this for KubeRay versions < 1.5.1, hence the manual version